flink-table/flink-table-api-scala-bridge/src/main/scala/org/apache/flink/table/api/bridge/scala/internal/StreamTableEnvironmentImpl.scala

```diff
@@ -237,17 +236,17 @@ class StreamTableEnvironmentImpl (
     Preconditions.checkNotNull(targetSchema, "Target schema must not be null.")
     Preconditions.checkNotNull(changelogMode, "Changelog mode must not be null.")
 
-    val schemaTranslationResult = ExternalSchemaTranslator.fromInternal(
+    val schemaTranslationResult = SchemaTranslator.createProducingResult(
       table.getResolvedSchema,
       targetSchema)
 
     toStreamInternal(table, schemaTranslationResult, changelogMode)
   }
 
   private def toStreamInternal[T](
-      table: Table,
-      schemaTranslationResult: ExternalSchemaTranslator.OutputResult,
-      @Nullable changelogMode: ChangelogMode)
+                                   table: Table,
```

Comment:
please do a manual pass over all changes again, esp. for Scala the code style sometimes might be messed up after refactorings